client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -39,6 +39,24 @@ export const useRequestColumns = () => {
         getSortValue: rowData => rowData.item.name,
       },
     ],
+    {
+      key: 'unit',
+      label: 'label.unit',
+      align: ColumnAlign.Left,
+
+      // width: 100,
```

Comment:
These widths don't seem to do anything here. In fact, I commented out all the widths in all the column definitions and the table looked exactly the same. Is something broken? Do we need to change something?

Reply:
yeah, we need to use `minWidth` to force the width. Could probably just remove the width prop from the cells to avoid confusion. but not now..

Reply:
So just leave it as it for now?

Reply:
I think so. Could remove the widths from this file to make it neater, but no need to